sps-team-38/src/main/webapp/script.js

```diff
@@ -53,3 +53,51 @@ function addInput(divName){
     }
 }   
 
+const formListElement = document.getElementById('form-list');
+const list = document.querySelector('#form-list ul');
+  // Load existing forms in database
+function loadForms(){
+    fetch('/list-forms').then(response => response.json()).then((forms) => {
+    forms.forEach((form) => {
+        formListElement.appendChild(renderForm(form));
+    })
+    });
+}
+// ................remove this section, just for reference....................
```

Comment:
Is "this section" referring to renderForm? If so, why is it part of the PR? If not, and if this is relating to something already removed, I think we should remove this comment to avoid confusion down the line.

Reply:
Right this was a reminder to my future self to make changes to this section. Will remove this. Thanks